packages/material-ui/test/Form.test.tsx

```diff
@@ -212,3 +212,43 @@ describe("single fields", () => {
     expect(tree).toMatchSnapshot();
   });
 });
+
+describe("refs", () => {
+  test("Renders with a ref and the Form Element accepts a generic formData argument", () => {  
+    let ref = React.createRef<CoreForm<{
+      field: string;
+    }>>()
+ 
+    renderer.create(
+      <Form<{
+        field: string;
+      }>
+        ref={ref}
+        formData={{
+          field: "",
+        }}
+        schema={{
+          type: "object",
+          properties: {
+            field: {
+              type: "string",
+            },
+          },
+        }}
+      />
+    );
+
+    expect(ref.current).toBeDefined();
+    if (ref.current) {
```

Comment:
I think it would be simpler just to do, instead of the else statement,

`expect(ref.current).toBeTruthy() // Expect ref to be properly forwarded`